engine/rpcserver.go

```diff
@@ -2118,12 +2115,7 @@ func (s *RPCServer) GetExchangeOrderbookStream(r *gctrpc.GetExchangeOrderbookStr
 			return errDispatchSystem
 		}
 
-		d := *data.(*interface{})
-		if d == nil {
-			return errors.New("unable to type assert data")
-		}
-
-		ob, ok := d.(orderbook.Base)
+		ob, ok := data.(orderbook.Base)
```

Comment:
go run ./... getexchangeorderbookstream -exchange="btc markets"
2022/04/14 12:23:36 rpc error: code = Unknown desc = unable to type assert orderbook data